### DISCUSSION TITLE
Suggestions for documentation/knowledge management platforms with Plausible integration

### DISCUSSION THREAD

**Discussion by leonvh4** [plausible/analytics#3307] [Anything Else]
Hi Plausible community - Does anyone have suggestions for documentation/knowledge management platforms that have great Plausible integrations? We host our docs site on Gitbook, and while it has a Plausible integration to get all the analytics, it does not allow for the tracking of goals, funnels, and outbound clicks as one cannot edit the tracking snippet. Does anyone have experience with similar platforms that have an integration with Plausible (or that allows for the adding of custom code) where you can get the full Plausible functionality? Thanks!

**Comment by metmarkosaric** [comment]
hi @leonvh4! perhaps Gitbook would be able to update their integration to [support our extensions](https://plausible.io/docs/script-extensions#all-our-script-extensions)? that may be the best approach so you don't need to switch to a different docs platform. for our docs, we use Docusaurus but there's no automatic plugin for this either. we just manually insert our snippet into the header of the docs.

**Reply by leonvh4** [reply]
Hey @metmarkosaric thanks for the response. Yeah I did reach out to them, and they noted it as a feature request for the future.
